identity-wallet/src/state/connections/mod.rs

```diff
@@ -21,3 +21,10 @@ pub struct Connection {
 
 #[typetag::serde(name = "connection")]
 impl FeatTrait for Connection {}
+
+impl Connection {
+    // TODO(ngdil): Temporary solution to support NGDIL demo, replace with different unique identifier to distinguish connection
+    pub fn create_connection_id(client_name: &str) -> String {
+        base64::encode_config(&client_name, base64::URL_SAFE)
```

Comment:
We now use this method for both `client_name`'s and `issuer_name`'s. To avoid confusion I think it's slightly better to go for the more generic `connection_name` as variable name her (instead of `client_name`.